README.md

```diff
@@ -28,9 +28,9 @@ The complete list of FIWARE GEs and Incubated FIWARE GEs can be found in the [FI
 
 **NGSI-LD Context Broker Feature Comparison**
 
-The NGSI-LD Specification is regularly updated published by ETSI. The latest specification is [version 1.7.1 ](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.07.01_60/gs_cim009v010701p.pdf) which was  published in June 2023. 
+The NGSI-LD Specification is regularly updated published by ETSI. The latest specification is [version 1.8.1](https://www.etsi.org/deliver/etsi_gs/CIM/001_099/009/01.08.01_60/gs_CIM009v010801p.pdf) which was  published in April 2024.
```

Comment:
I think an _and_ should be added here :  `The NGSI-LD Specification is regularly updated **and** published`

Reply:
good catch. fixed.